modules/orchestrator/services/addon/addon.go

```diff
@@ -1401,7 +1401,7 @@ func (a *Addon) Delete(userID, routingInstanceID string) error {
 			return err
 		}
 		if !permissionResult.Access {
-			return errors.New("权限不足")
+			return errors.New("access denied")
```

Comment:
it's not i18n, it just replace chinese to english?

Reply:
yes, the err by errors.new just replace chinese to english
![image](https://user-images.githubusercontent.com/23724009/130200977-922ce0ad-9c4d-4161-ad03-7652f46b2eee.png)